src/components/AttentionBox/AttentionBox.module.scss

```diff
@@ -9,7 +8,7 @@
   width: fit-content;
   max-width: 100%;
   height: fit-content;
-  @include theme-prop(color, dark-color);
+  color: var(--color-mud_black);
```

Comment:
why changing from dark to mud black?

Reply:
<img width="633" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/aad89e7d-ec15-408e-ae16-61a61bd4399b">
<img width="716" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/b388d3a8-10c0-4d92-8520-4c0857e2d453">